content/en/golden_signals.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Golden Signals
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+[Golden signals](https://sre.google/sre-book/monitoring-distributed-systems/#xref_monitoring_golden-signals) are metrics that help us understand how [services](https://glossary.cncf.io/service/) are experienced by users. Today, it is the gold standard for monitoring application performance and focuses on four specific metrics: Latency, traffic, errors, and saturation. The first three metrics relate to requests. A request is a message sent from one service to another. Services do that to collaborate and work as a [distributed app](https://glossary.cncf.io/distributed_apps/).
+
+*Latency* is the time it takes to service a request. *Traffic* measures the request frequency placed on the application. *Errors* measure the request failure rate. And *Saturation* measures the capacity of app resources, allowing users to address bottlenecks before they become a problem.
+ 
+## Problem it addresses
+Without metrics, engineers have nothing to deduce why applications are behaving a certain way. While earlier monitoring tools were designed to monitor machines (not applications/services), modern monitoring systems gather a huge variety of parameters. They collect so many metrics that it's easy to lose focus on what is relevant for the user. Which of all these metrics should you focus on to ensure your customers have a great user experience?
+
+## How it helps
+Golden signals help engineers focus on metrics that matter for the user experience. They help steer clear from getting stuck in the mud by putting performance data into different buckets and providing the visibility needed to pinpoint problems that might affect the user.
+
+*Latency* tells us how long it takes the application to handle transactions. An increase in latency is likely a sign of a performance issue.
+
+Measuring website traffic allows a clear understanding of what is working or what isn't and develops a more potent content. It helps in better utilization of traffic.
```

Comment:
Don't think this is the right example. It's probably more about knowing when to app should scale and balance the load? But I'd defer to @iamNoah1.

Reply:
I agree. If the traffic increases, we should scale up and the other way around. I am still not sure if we really need to explain what those metrics are.